examples/php/composer.json

```diff
@@ -7,7 +7,9 @@
   },
   "autoload": {
     "psr-4": {
-      "": "route_guide/"
+      "": "route_guide/",
+      "GPBMetadata\\": "GPBMetadata/",
```

Comment:
I believe you don't need this line. Please try removing it.

Reply:
I got class 'GPBMetadata\Helloworld' not found, if remove this line